design-layout/css-layout-position-assignmnet-3/dashboard-design-layout-assignment-3.1/dashbord-layout.html

```diff
@@ -0,0 +1,274 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
```

Comment:
Reason to use this tag?

Reply:
to support all standard mode in   Internet Explorer  but it is not  necessary to use